pyreisejl/utility/app.py

```diff
@@ -26,11 +26,8 @@ def get_script_path():
     return str(path_to_script)
 
 
-@app.route("/launch/<int:scenario_id>", methods=["POST"])
-def launch_simulation(scenario_id):
+def launch_simulation(scenario_id, threads=None, solver=None):
```

Comment:
If we're going to be using this to launch simulations from `powersimdata`, should we add a check for available solvers here just so that it fails faster? i.e. `from pyreisejl.utility.launchers import get_available_solvers`?